<launch>
    <!--打开手柄控制节点-->
    <node pkg="wheeltec_joy" type="ptz_control_xbox" name="ptz_control_xbox" output="screen">
       <param name="axis_x"   type="int"      value="0" />
       <param name="axis_y"  type="int"      value="1"/>
       <param name="axis_fast_x"       type="int"   value="3" />
       <param name="axis_fast_y"      type="int"   value="4"/>
       <param name="vlinear_x"   type="double"      value="32" />
       <param name="vlinear_y"  type="double"      value="32"/>
       <param name="vlinear_fast_x"       type="double"   value="63" />
       <param name="vlinear_fast_y"      type="double"   value="63"/>
    </node>
    <!--获取手柄数据节点-->
   <include file="$(find wheeltec_joy)/launch/joy_data.launch" />
</launch>
